### PR TITLE
docs(eol): Add EOL options text and link to LTS section of landing page

### PR DESCRIPTION
**Pull request by edclement**
# AngularJS Material is in LTS mode

We are no longer accepting changes that are not critical bug fixes into this project.
See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.

## PR Checklist

Please check your PR fulfills the following requirements:
- [ ] Does this PR fix a regression since 1.2.0, a security flaw, or a problem caused by a new browser version?
- [x] The commit message follows [our guidelines](https://github.com/angular/material/blob/master/.github/CONTRIBUTING.md#-commit-message-format)
- [x] Tests for the changes have been added or this is not a bug fix / enhancement
- [x] Docs have been added, updated, or were not required

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Enhancement
[x] Documentation content changes
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
image

## What is the new behavior?
image

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/app/partials/home.tmpl.html [hunk 1]
```diff
@@ -289,4 +290,3 @@ <h2 class="md-headline">Conference Presentations:</h2>
     </p>
   </md-content>
 </div>
-
```
Comment (Splaktar) [inline docs/app/partials/home.tmpl.html:292]: The blank line at the end of the file should remain.

File: docs/app/partials/home.tmpl.html [hunk 2]
```diff
@@ -63,7 +63,8 @@ <h2 id="long-term-support" class="md-headline">Long Term Support</h2>
       AngularJS Material has entered a Long Term Support (LTS) period which ends on December 31st,
       2021. This date aligns with the
       <a href="https://docs.angularjs.org/misc/version-support-status" target="_blank" rel="noopener">
-        AngularJS LTS</a> period.
+      AngularJS LTS</a> period. Explore End Of Life (EOL) options
+      <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
```
Comment (Splaktar) [inline docs/app/partials/home.tmpl.html:67]:
```suggestion
      <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429" target="_blank" rel="noopener">here</a>.
```
This should use these attributes similar to the link above.

**Review by Splaktar — CHANGES_REQUESTED** [review]

File: docs/app/partials/home.tmpl.html (hunk 1 above)
Comment (edclement) [reply docs/app/partials/home.tmpl.html:292]: This must have been IntelliIJ... I'll update. Thanks.

**Review by Splaktar — APPROVED** [review]
LGTM

**Review by mgechev — APPROVED** [review]

**Pull request merged**